src/Storages/System/InformationSchema/tables.sql

```diff
@@ -1,17 +1,35 @@
 ATTACH VIEW tables
-(
-    `table_catalog` String,
-    `table_schema` String,
-    `table_name` String,
-    `table_type` Enum8('BASE TABLE' = 1, 'VIEW' = 2, 'FOREIGN TABLE' = 3, 'LOCAL TEMPORARY' = 4, 'SYSTEM VIEW' = 5),
-    `TABLE_CATALOG` String ALIAS table_catalog,
-    `TABLE_SCHEMA` String ALIAS table_schema,
-    `TABLE_NAME` String ALIAS table_name,
-    `TABLE_TYPE` Enum8('BASE TABLE' = 1, 'VIEW' = 2, 'FOREIGN TABLE' = 3, 'LOCAL TEMPORARY' = 4, 'SYSTEM VIEW' = 5) ALIAS table_type
-) AS
-SELECT
-    database AS table_catalog,
-    database AS table_schema,
-    name AS table_name,
-    multiIf(is_temporary, 4, engine like '%View', 2, engine LIKE 'System%', 5, has_own_data = 0, 3, 1) AS table_type
-FROM system.tables
+    (
+     `table_catalog` String,
+     `table_schema` String,
+     `table_name` String,
+     `table_type` String,
+     `table_comment` String,
+     `table_collation` String,
+     `TABLE_CATALOG` String,
+     `TABLE_SCHEMA` String,
+     `TABLE_NAME` String,
+     `TABLE_TYPE` String,
+     `TABLE_COMMENT` String,
+     `TABLE_COLLATION` String
+        ) AS
+SELECT database             AS `table_catalog`,
+       database             AS `table_schema`,
+       name                 AS `table_name`,
+       comment              AS `table_comment`,
```

Comment:
Thoughts:
- Looking at [MySQL docs](https://dev.mysql.com/doc/refman/8.0/en/information-schema-tables-table.html), their version of `INFORMATION_SCHEMA.TABLES` includes many more fields. Your PR merely adds `table_comment` and `table_collation`. Wondering that Tableau Online does not complain while there are still missing fields?
- Let's keep fields `comment` and `table_type` in l. 6/7 and l. 19-26 in the same order.
- Please don't forget to document the new fields [here](https://clickhouse.com/docs/en/operations/system-tables/information_schema#columns), respectively update the type of existing fields.
- Same issue exists with alias expressions vs. duplicated columns as in `INFORMATION_SCHEMA.SCHEMATA`.
- Hardcoding the table collation as `utf8mb4_0900_ai_ci` is weird. 1. ClickHouse has no per-column/per-table/per-schema collations. We can surely hardcode something Unicode-like but then let's display something more generic, e.g. just `utf8` - assuming Tableau Online does not expect exactly `utf8mb4_0900_ai_ci`.